infra/cmake/packages/Egl_HeadersSourceConfig.cmake

```diff
@@ -0,0 +1,21 @@
+function(_Egl_HeadersSource_import)
```

Comment:
please split infra in separate PR

Reply:
Yes, I will split it.

Reply:
I splited through (https://github.com/Samsung/ONE/pull/9257) . please check it. If the it is merged, i will proceed with the correction of this PR.